platform/ios/src/MGLCalloutView.h

```diff
@@ -36,6 +36,18 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, weak) id<MGLCalloutViewDelegate> delegate;
 
 /**
+ A Boolean value indicating whether the callout view should be positioned automatically.
+ The exact position can be adjusted by overriding -[UIView setCenter:]
+ */
+@property (nonatomic, assign) BOOL automaticPositioning;
```

Comment:
A Boolean-typed property shouldn’t be named with a noun phrase. How about `anchoredToAnnotation`, with a getter `isAnchoredToAnnotation`?